DNN Platform/Website/DotNetNuke.Website.csproj

```diff
@@ -3345,6 +3345,7 @@
     <Content Include="Providers\DataProviders\SqlDataProvider\09.05.01.SqlDataProvider" />
     <Content Include="Providers\DataProviders\SqlDataProvider\09.06.01.SqlDataProvider" />
     <Content Include="Providers\DataProviders\SqlDataProvider\09.06.02.SqlDataProvider" />
+    <Content Include="Providers\DataProviders\SqlDataProvider\09.07.00.SqlDataProvider" />

```

Comment:
9.7.0 is currently in the RC stage, so this change would go in 9.7.1.  Can you rename the `SqlDataProvider` file to `09.07.01`, please?

Reply:
Done